src/mce/provider_mce.cpp

```diff
@@ -87,9 +88,9 @@ void Bridge::init()
 }
 
 
-MceNs::MceNs(QDBusConnection &bus, std::shared_ptr<ScreenNs> const &screen)
-    : Namespace("System", std::unique_ptr<PropertiesSource>
-                (new Bridge(this, bus)))
+MceNs::MceNs(QDBusConnection &bus, std::shared_ptr<ScreenNs> const &screen
+             , statefs_provider_mode mode)
```

Comment:
no, this is my style of parameter list continuation. IMO, it is more logical because ```,``` purpose is it to separate the next parameter, it should be put only if the parameter exists.